packages/cli/lib/generate-ts/dbMetaToTsMapper.ts

```diff
@@ -0,0 +1,55 @@
+import * as _ from "lodash";
+import { pascal } from "change-case";
+
+import { IDbMeta, IColumnType, IEnum, ITable, IColumn, IConstraint } from "@fullstack-one/schema-builder";
+
+export function mapDbMetaToTypeDefinitions(dbMeta: IDbMeta): string {
+  const tsEnums: string = mapDbMetaEnumsToTypescriptEnums(dbMeta.enums);
+  const tsInterfaces: string = mapDbMetaTablesToTypescriptInterfaces(dbMeta.schemas.public.tables, _.map(dbMeta.enums, (value, key) => key));
+
+  return `${tsEnums}\n\n${tsInterfaces}\n`;
+}
+
+function mapDbMetaEnumsToTypescriptEnums(dbMetaEnums: { [name: string]: IEnum }): string {
+  return _.map(dbMetaEnums, (enumItem) => {
+    const values: string = _.map(enumItem.values, (value) => `  ${pascal(value)} = "${value}"`).join(",\n");
+    return `export enum ${enumItem.name} {\n${values}\n}`;
+  }).join("\n\n");
+}
+
+function mapDbMetaTablesToTypescriptInterfaces(dbMetaTables: { [name: string]: ITable }, enums: string[] = []): string {
+  return _.map(dbMetaTables, (table) => {
+    const constraints = table.constraints;
+    const tsProperties: string = _.map(table.columns, (column) => mapDbMetaColumnToTypescriptProperty(column, constraints, enums)).join("\n");
```

Comment:
map is nativelly supportet by ES: 
e.g.
`Object.values(table.columns).map(column => {...})`